pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -451,8 +438,34 @@ def compile(
                 replace_libs=replace_libs,
                 replay_from=replay_from,
             )
-        install_for_distribution()
-    del bash_runner.env["_PYTHON_HOST_PLATFORM"]
+
+
+def set_host_platform(wheel_dir: Path, platform: str):
+    dist_info = next(wheel_dir.glob("*.dist-info"))
+    WHEEL = dist_info / "WHEEL"
+    lines = WHEEL.read_text().splitlines()
+    result_lines = []
+    for line in lines:
+        if line.startswith("Tag:"):
+            line = line.rpartition("-")[0] + "-" + platform
+        result_lines.append(line)
+    WHEEL.write_text("\n".join(result_lines))
+
+
+def update_wheel_platform(build_dir: Path, wheel_dir: Path):
```

Comment:
Would you mind adding a docstring explaing what exactly we are updating and where?

Reply:
I think this function was removed in #2256.